Source: http://websvn.kde.org/trunk/extragear/base/polkit-kde-1/agent/AuthDialog.cpp?r1=1065570&r2=1073600
Upstream: yes
Reason: Build with gcc 4.5.0; fixes direct call to contructor of KUser

--- trunk/extragear/base/polkit-kde-1/agent/AuthDialog.cpp	2009/12/23 18:00:55	1065570
+++ trunk/extragear/base/polkit-kde-1/agent/AuthDialog.cpp	2010/01/12 13:57:17	1073600
@@ -195,7 +195,7 @@
         foreach(PolkitQt1::Identity *identity, identities) {
             // First check to see if the user is valid
             qDebug() << "User: " << identity;
-            KUser user = KUser::KUser(identity->toString().remove("unix-user:"));
+            KUser user(identity->toString().remove("unix-user:"));
             if (!user.isValid()) {
                 kWarning() << "User invalid: " << user.loginName();
                 continue;
